iteration.go

```diff
@@ -233,6 +262,39 @@ func createIterationLinks(request *goa.RequestData, id interface{}) *app.Generic
 	}
 }
 
+// ConvertFromLtreeFormat converts data to UUID format from ltree format.
+func ConvertFromLtreeFormat(uuid string) string {
+	// Ltree allows only "_" as a special character.
+	return strings.Replace(uuid, "_", "-", -1)
+}
+
+// IterationIDMap contains a map that will hold iteration's ID as its key
+type IterationIDMap map[uuid.UUID]*iteration.Iteration
```

Comment:
good note, thanks now it is un-exported https://github.com/almighty/almighty-core/pull/713/commits/f3703a067de6eb90ae312e206834df8bb4b471a7